client/src/pages/Test.js

```diff
@@ -25,6 +27,7 @@ function Test() {
     <>
       <button onClick={handleClick}> here</button>
       <LogoutBtn />
+      <GoogleLogout/>
```

Comment:
@Adi-tya93  there are 2 btns for logout, the simple one only signs out the user from the app, next time u log in, it won't ask for consent, and u wouldn't receive refrsh_token.  the second btn with google logo, it will signout the user from app, as well as disconnect the account. So next time u log in, it will send u refresh token. 

for our app, we will only use the first btn (sign out from the app, but wouldn't revoke the refresh token).